js_modules/dagit/packages/core/src/runs/RunsFilterInput.tsx

```diff
@@ -171,10 +200,43 @@ export const RunsFilterInput: React.FC<RunsFilterInputProps> = ({
   enabledFilters,
 }) => {
   const {options} = useRepositoryOptions();
+  const [selectedTagKey, setSelectedTagKey] = React.useState<string | undefined>();
+  const [fetchTagKeys, {data: tagKeyData}] = useLazyQuery<RunTagKeysQuery>(RUN_TAG_KEYS_QUERY, {
+    fetchPolicy: 'cache-and-network',
```

Comment:
This is already our default fetch policy, you don't need to add it to either of the queries here.